### DISCUSSION TITLE
Key toggle `&kt` not supported by UI?

### DISCUSSION THREAD

**Discussion by Richiban** [nickcoutsos/keymap-editor#59] [Q&A] [answered]
It seems that key toggle isn't supported by the tool? It doesn't appear in the behaviour drop-down and if `&kt` appears in the keymap file the UI displays an error.

**Comment by nickcoutsos** [comment, marked as answer]
Hmm, you're right!

Currently behaviours are manually added, something I want to eventually be able to automate at least a little, so it's easy for me to overlook some behaviours. I'll try to address this quickly as its a trivial addition.
